test_dumpspecs broken under python 3

The `<skipped>` must have been an accidental copy/paste on my part. 

This would have never passed under Python 3.6 anyway, so we need to sanitize `python ver = ` a little further under Python 3.

```python
tests/test_describe.py:29: AssertionError
________________________________ test_dumpspecs ________________________________

    @pytest.mark.skipif(not HAS_IRAF, reason='Need IRAF to run')
    def test_dumpspecs():
        # get dumpspecs output
        out_f = six.StringIO()
        wutil.dumpspecs(outstream=out_f, skip_volatiles=True)
        out_str = out_f.getvalue()
        out_f.close()
    
        # modify out_str to remove a path that will always be changing
        out_str = '\n'.join([l for l in out_str.split('\n')
                             if 'python exec =' not in l])
        # modify out_str to handle old versions which printed Tkinter as camel-case
        out_str = out_str.replace('Tkinter', 'tkinter')
    
        # verify it (is version dependent)
        key = ('2' if IS_PY2 else '3', sys.platform.replace('2', ''))
        expected = REF[key]
>       assert expected.strip() == out_str.strip(), \
            'Unexpected output from wutil.dumpspecs: {}'.format(out_str)
E       AssertionError: Unexpected output from wutil.dumpspecs: python ver = 3.6
E         platform = linux
E         PY3K = True
E         c.OF_GRAPHICS = False
E         /dev/console owner = <skipped>
E         tkinter use unattempted.
E         
E       assert 'python ver =... unattempted.' == 'python ver = ... unattempted.'
E         - python ver = 3<skipped>
E         + python ver = 3.6
E           platform = linux
E           PY3K = True
E           c.OF_GRAPHICS = False
E           /dev/console owner = <skipped>
E           tkinter use unattempted.

tests/test_graphics.py:194: AssertionError```

--- a/tests/test_graphics.py
+++ b/tests/test_graphics.py
@@ -56,7 +56,7 @@ tkinter use unattempted.
 """
     REF[('2', 'darwin')] = REF[('2', 'linux')].replace('linux2', 'darwin')
 
-    REF[('3', 'linux')] = """python ver = 3<skipped>
+    REF[('3', 'linux')] = """python ver = 3.6
 platform = linux
 PY3K = True
 c.OF_GRAPHICS = False

